src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 name="hederlink" >Moyo header</h1>
+    <header class="header">
+      <a href="#" id="hederlink"
+      class="logo">
```

Comment:
```suggestion
      <a 
        href="#"
        id="headerlink"
        class="logo"
      >
```
Если у тега больше двух атрибутов - переноси каждый на новую линию